source/quick-start-rails/view-data.txt

```diff
@@ -0,0 +1,90 @@
+.. _mongoid-quick-start-rails-view-data:
+
+=================
+View MongoDB Data
+=================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a data model
+
+      Run the following command from your project root to create a
+      ``Restaurant`` model with ``name``, ``cuisine``, and ``borough``
+      fields:
+
+      .. code-block:: bash
+
+         bin/rails g scaffold Restaurant name:string cuisine:string borough:string
+      
+      This command also creates the controller and view files for the
+      ``Restaurant`` model. You can find the directories that contain
+      these files in the ``app`` directory of your application.
+
+   .. step:: Retrieve specific documents
+
+      The ``app/controllers/restaurants_controller.rb`` file contains
+      definable methods that specify how your app handles different
+      requests. Replace the ``index()`` method with the following code:
+
+      .. code-block:: ruby
+
+         def index
+           @restaurants = Restaurant
+               .where(name: /earth/i)
+         end
+
+      This controller method retrieves restaurant documents in which the
+      value of the ``name`` field contains the string ``"earth"``. The
+      results are rendered at the ``/restaurants`` route by default.
+
+   .. step:: Start your Rails application
+
+      Run the following command from the application root directory
+      to start your {+language+} web server:
+
+      .. code-block:: bash
+
+         bin/rails s
+
+      After the server starts, it outputs the following message
+      indicating that the application is running on port ``3000``:
+
+      .. code-block:: none
+         :copyable: false
+
+         => Booting Puma
+         => Rails 7.2.1 application starting in development 
+         => Run `bin/rails server --help` for more startup options
+         Puma starting in single mode...
+         * Puma version: 6.4.3 (ruby 3.2.5-p208) ("The Eagle of Durango")
+         *  Min threads: 3
+         *  Max threads: 3
+         *  Environment: development
+         *          PID: 66973
+         * Listening on http://127.0.0.1:3000
+         * Listening on http://[::1]:3000
+         * Listening on http://127.0.2.2:3000
+         * Listening on http://127.0.2.3:3000
+         Use Ctrl-C to stop
+
+   .. step:: View the restaurant data
+
+      Open the URL http://127.0.2.2:3000/restaurants in your web browser.
+      The page shows a list of restaurants and details about each of
+      them:
+      
+      .. figure:: /includes/figures/quickstart-rails-list.png
+         :alt: The rendered list of restaurants
+
+      Rails provides a default interface that allows you to view, edit,
+      and delete models.
```

Comment:
Q: Does the previous URL show this interface? If not, I don't see the connection to this idea

Edit: Based on the next page, looks like it does have an interface. Maybe add some connection to this paragraph: e.g., "See the next section to learn how to use the interface to edit MongoDB data."